cropper/app/lib/Permissions.scala

```diff
@@ -0,0 +1,30 @@
+package lib
+
+import scala.concurrent.{ExecutionContext, Future}
+import com.gu.mediaservice.lib.auth._
+
+
+object PermissionDeniedError extends Throwable("Permission denied")
+
+object Permissions {
```

Comment:
We can hopefully use this to do things like multiple permission etc later... probably move it over to common when / if we need to.